core/domain/wipeout_service.py

```diff
@@ -66,14 +66,14 @@
     exp_models, feedback_models, improvements_models,
     question_models, skill_models, story_models,
     subtopic_models, suggestion_models, topic_models,
-    user_models
+    user_models, opportunity_models
 ) = models.Registry.import_models([
     models.Names.APP_FEEDBACK_REPORT, models.Names.BASE_MODEL,
     models.Names.BLOG, models.Names.COLLECTION, models.Names.CONFIG,
     models.Names.EXPLORATION, models.Names.FEEDBACK, models.Names.IMPROVEMENTS,
     models.Names.QUESTION, models.Names.SKILL, models.Names.STORY,
     models.Names.SUBTOPIC, models.Names.SUGGESTION, models.Names.TOPIC,
-    models.Names.USER,
+    models.Names.USER, models.Names.OPPORTUNITY
```

Comment:
I'm not sure about this, stuff in OPPORTUNITY is typically more related to exploration/skill data than a user. I left a suggestion below to move the storage model into UserModels since the new model you are adding is really a user preference.

Reply:
Shifted PinnedOpportunityModel to user models.